### ISSUE TITLE
Tqdm bar has unexpected behavior when val_check_interval < 1

### ISSUE THREAD

**Issue by RomainSabathe** [williamFalcon/pytorch-lightning#232]
**Describe the bug**
If we set the trainer to have val_check_interval < 1, then here's what happens:

* From batch_nb = 0 up to batch_nb = val_check_batch + 1: the tqdm bar correctly shows the progress on the training set.
* Onwards: the validation step begins. Every batch gathered from the validation set is counted and *added to the progress bar*. So the progress bar will continue from `val_check_batch + 1` up until `val_check_batch + 1 + len(val_dataloader)`. 

This is very counter intuitive and I suspect we can classify this as a bug! A more expected behavior would be to have a second tqdm bar spawning while the original training tqdm bar remains visible. 

**To Reproduce**
1. Use show_progress_bar=True
2. Set val_check_batch to a value strictly below 1. The close to 0, the easier it will be to observe the bug.

Thanks! If I have a bit of time over the weekend, I'd be happy to try and make a PR.

**Comment by williamFalcon** [comment]
that's actually done on purpose. the tqdm bar calculates a time estimate for the full loop. If you have only the training samples in there, you won't ever get an estimate of how long the full train+val loop will take.
